srcpkgs/oneko/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'oneko'
+pkgname=oneko
+version=1.2.5
+revision=1
+# other archs end with /usr/bin/xmkmf: line 53: 15231 Aborted
+archs="x86_64"
+_pkgver=1.2.sakura.5
+wrksrc=${pkgname}-${_pkgver}
+build_style=gnu-configure
+hostmakedepends="imake gccmakedep"
+makedepends="libX11-devel libXext-devel"
+short_desc="Cat that chases around your cursor"
+maintainer="wael <40663@protonmail.com>"
+license="Public Domain"
+homepage="http://www.daidouji.com/oneko"
+distfiles="${homepage}/distfiles/${pkgname}-${_pkgver}.tar.gz"
```

Comment:
expand `homepage` and `pkgname` here

Reply:
what do you mean by that? do you mean to use `$homepage` instead of `${homepage}` ?

Reply:
use the value instead of the variable, so just oneko instead of $pkgname

Reply:
i don't see why i should do that but it IS shorter so.

Reply:
it makes it so that if something changes, the change doesn't affect other places in the template, and people who use scripts to do stuff with the repository en masse don't have issues

Reply:
Please also expand $homepage